keyboards/mykeyclub/readme.md

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
```suggestion
```

Reply:
I'm new to pull requests. If I press "commit suggestion" are the changes made to the sizezero:jris65 branch?

Reply:
I feel like I should test all of these first before committing them.